src/Symfony/Component/Uid/Uuid.php

```diff
@@ -54,7 +54,9 @@ public static function fromString(string $uuid): parent
             $uuid = substr_replace($uuid, '-', 18, 0);
             $uuid = substr_replace($uuid, '-', 23, 0);
         } elseif (26 === \strlen($uuid) && Ulid::isValid($uuid)) {
-            $uuid = (new Ulid($uuid))->toRfc4122();
+            $ulid = new Ulid('00000000000000000000000000');
+            $ulid->uid = strtr($uuid, 'abcdefghjkmnpqrstvwxyz', 'ABCDEFGHJKMNPQRSTVWXYZ');
```

Comment:
can you do a quick bench on PHP8: wouldn't `strtoupper()` be faster? if yes, let's use it everywhere in the `Ulid::__construct()` also

Reply:
It's faster! :clap: